src/scripts/main.js

```diff
@@ -1 +1,77 @@
 'use strict';
+
+let clicks = false;
+
+function createNotification(message) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.textContent = message;
+
+  if (message.includes('resolved')) {
+    div.classList.add('success');
+  }
+
+  if (message.includes('rejected')) {
+    div.classList.add('error');
+  }
+
+  document.body.appendChild(div);
+}
+
+const p1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    clicks = true;
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    if (clicks === false) {
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
+});
+
+p1.then((message) => {
+  createNotification(message);
+}).catch((error) => {
+  createNotification(error.message);
+});
+
+const p2 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    resolve('Second promise was resolved');
+  });
```

Comment:
The 'contextmenu' event listener prevents the default context menu from appearing. Ensure this behavior is intended, as it might affect user experience.